core/src/main/java/org/elasticsearch/ingest/core/CompoundProcessor.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
+//TODO(simonw): can all these classes go into org.elasticsearch.ingest?
```

Comment:
Maybe for now we should move things into one package. When we want to make ingest a separate jar then this will be a separate project. There are other things that we will have to address  once we move to a dedicated ingest jar.